About/Knowledge-base.md

```diff
@@ -61,6 +61,13 @@ Inside zipped file you will find:
 
 ### Demos
 
+##### Building on top of Golem with gWASM
+Have you missed our workshop at Devcon5 in Osaka? Fear not! We're pleased to give you the very workshop re-recorded at our office at the heart of Poland in Warsaw. Learn about Golem, our meta-use case gWasm, and once you digested all the information... please remember to put what you learn into good use and start coding for gWasm!
```

Comment:
```suggestion
Did you miss our workshop at Devcon5 in Osaka? Fear not! We're pleased to give you the very workshop re-recorded at our office at the heart of Poland in Warsaw. Learn about Golem, our meta-use case gWasm, and once you have digested all the information...put what you learned to good use and start coding for gWasm!
```

Reply:
It was actually copied from the video description. Do you recommend updating it as well?